seed/seed.json

```diff
@@ -1224,6 +1224,7 @@
             ],
             "filter": {
                 "min_version": "104.1.43.54",
+                "max_version": "106.1.45.44",
```

Comment:
Here is the proof that v1.45.44 has the commit to enable the feature:
https://github.com/brave/brave-core/compare/v1.45.43...v1.45.44

I've set 106, not 105 because the final 1.45.x builds use cr106.